api/swagger-spec/root.json

```diff
@@ -0,0 +1,54 @@
+{
+  "swaggerVersion": "1.2",
+  "apis": [
+   {
+    "path": "/api",
+    "description": "get available API versions"
+   },
+   {
+    "path": "/api/v1",
+    "description": "API at /api/v1 version v1"
+   },
+   {
+    "path": "/api/v1beta1",
+    "description": "API at /api/v1beta1 version v1beta1"
+   },
+   {
+    "path": "/api/v1beta2",
+    "description": "API at /api/v1beta2 version v1beta2"
+   },
+   {
+    "path": "/api/v1beta3",
+    "description": "API at /api/v1beta3 version v1beta3"
+   },
+   {
+    "path": "/oapi",
+    "description": ""
+   },
+   {
+    "path": "/oapi/v1",
+    "description": "OpenShift REST API, version v1"
+   },
+   {
+    "path": "/osapi",
+    "description": ""
+   },
+   {
+    "path": "/osapi/v1beta1",
+    "description": "OpenShift REST API, version v1beta1"
+   },
+   {
+    "path": "/osapi/v1beta3",
+    "description": "OpenShift REST API, version v1beta3"
+   },
+   {
+    "path": "/version",
```

Comment:
Too painful right now - it has to be part of go-restful, and health is just a basic mux.

----- Original Message -----
> > +    "description": "OpenShift REST API, version v1"
> > +   },
> > +   {
> > +    "path": "/osapi",
> > +    "description": ""
> > +   },
> > +   {
> > +    "path": "/osapi/v1beta1",
> > +    "description": "OpenShift REST API, version v1beta1"
> > +   },
> > +   {
> > +    "path": "/osapi/v1beta3",
> > +    "description": "OpenShift REST API, version v1beta3"
> > +   },
> > +   {
> > +    "path": "/version",
> 
> Can we add `healthz` even though it's not served by go-restful or is that too
> painful?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2660/files#r31464696